roles/kubernetes/master/templates/kubeadm-config.v1beta2.yaml.j2

```diff
@@ -88,8 +88,14 @@ dns:
   imageTag: {{ coredns_image_tag }}
 networking:
   dnsDomain: {{ dns_domain }}
-  serviceSubnet: {{ kube_service_addresses }}
-  podSubnet: {{ kube_pods_subnet }}
+  serviceSubnet: "{{ kube_service_addresses }}{{ ',' + kube_service_addresses_ipv6 if enable_dual_stack_networks }}"
+  podSubnet: "{{ kube_pods_subnet }}{{ ',' + kube_pods_subnet_ipv6 if enable_dual_stack_networks }}"
+{% if kube_feature_gates %}
```

Comment:
You also globally add support fot featureGates ?
we have this options in api,cm and scheduler

also in kube-proxy https://github.com/kubernetes-sigs/kubespray/pull/6851

Reply:
Sure :)